pygmt/datasets/samples.py

```diff
@@ -22,6 +76,32 @@ def load_japan_quakes():
         The data table. Columns are year, month, day, latitude, longitude,
         depth (in km), and magnitude of the earthquakes.
     """
+
+    warnings.warn(
+        "This function has been deprecated since v0.6.0 and will be removed "
+        "in v0.9.0. Please use load_sample_dataframe(name='tut_quakes.ngdc') "
+        "instead.",
+        category=FutureWarning,
+        stacklevel=2,
+    )
+
+    return load_sample_dataframe("japan_quakes")
+
+
+def load_tut_quakes():
```

Comment:
If this function is meant to be removed at some point, maybe add an underscore to indicate it is non-user-facing
```suggestion
def _load_tut_quakes():
```